setup.py

```diff
@@ -47,5 +48,10 @@
         "Topic :: Utilities",
         "License :: OSI Approved :: Apache Software License",
     ],
-    entry_points={"console_scripts": ["neuro-flow=neuro_flow.cli:main"]},
+    entry_points={
+        "console_scripts": [
+            "neuro-flow=neuro_flow.cli:main",
+            "neuro-batch-executor=neuro_flow.batch_executor:entrypoint",
```

Comment:
I created new entry point, because `neuro-flow.cli:main` looks for config folder on main group level. I can add some code to make exception for executor command, but it is very ugly solution as for me.
Is there any way to hide this entry point from users? I tried to parse `sys.argv` in `setup.py`, but it doesn't work because setup files of our dependencies fail to parse my additional argument.

Reply:
Please replace the entrypoint with a hidden CLI command.